components/current-apprentices/current-apprentices.test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { CurrentApprentices } from './current-apprentices.js';
+
+describe('the CurrentApprentices component', () => {
+  let renderedImages;
+  let version;
+
+  beforeAll(() => {
+    const currentApprenticeClass = {
+      version: '14.0',
+      currentApprentices: [
+        {
+          name: 'John Doe',
+          image: '/apprentices/John.png',
+          links: [
+            { href: 'https://www.linkedin.com/', text: 'Linkedin' },
+            { href: 'personal', text: 'Personal' },
+            { href: 'https://www.github.com/', text: 'Github' },
+          ],
+        },
+        {
+          name: 'Jane Doe',
+          image: '/apprentices/Jane.png',
+          links: [
+            { href: 'https://www.linkedin.com/', text: 'Linkedin' },
+            { href: 'personal', text: 'Personal' },
+            { href: 'https://www.github.com/', text: 'Github' },
+          ],
+        },
+      ],
+    };
+
+    render(<CurrentApprentices currentApprenticeClass={currentApprenticeClass} />);
+
+    renderedImages = screen.getAllByRole('img');
+    version = screen.getByTestId('test-version-span');
```

Comment:
Was there a reason you went with the test id here, rather than something like this?

```
    version = screen.getByRole('heading', {
      name: /current apprentices/i,
    });
```

Reply:
No, I was actually trying to find a way to refrain from using the getByTestId, but didn't find a resolution in a timely manner. But I switched to use the above suggestion in the lates push.